### PR TITLE
add task solution

### PR DESCRIPTION
**Pull request by netzelnatalia**
[DEMO LINK](https://netzelnatalia.github.io/layout_calendar/)

### CODE REVIEW

File: src/styles/main.scss [hunk 1]
```diff
@@ -1,3 +1,57 @@
...
+    background-color: #eee;
+    box-sizing: border-box;
+    cursor: pointer;
+    transition: 0.5s;
+
+    @for $dayNumber from 1 through 31 {
+      &:nth-child(#{$dayNumber}):before {
+        content: "#{$dayNumber}";
+        font-size: 30px;
+        font-family: "Arial", sans-serif;
```
Comment (OleziO) [inline src/styles/main.scss]: Move to the html selector and remove it from all other places.

**Review by OleziO — CHANGES_REQUESTED** [review]
Please, deploy the project.

In terminal: `npm delpoy`

File: src/styles/main.scss [hunk 2]
```diff
@@ -1,3 +1,65 @@
...
   margin: 0;
 }
+html {
+  @for $dayNumber from 1 through 31 {
+    &:nth-child(#{$dayNumber}):before {
+      content: "#{$dayNumber}";
+      font-size: 30px;
+      font-family: "Arial", sans-serif;
+    }
+  }
```
Comment (volodymyr-soltys97) [inline src/styles/main.scss:22]: Move it ti calendar selector

**Review by volodymyr-soltys97 — CHANGES_REQUESTED** [review]
Need to make font size 30px for days number
image

File: src/styles/main.scss [hunk 3]
```diff
@@ -1,3 +1,67 @@
...
+      display: flex;
+      justify-content: flex-start;
+      list-style: none;
+      align-content: center;
+      flex-wrap: wrap;
+      gap: $gap;
+      width: $calendarSize;
+      padding: 10px;
+    }
+  }
```
Comment (etojeDenys) [inline src/styles/main.scss:22]: why do we need two curly brackets?

File: src/styles/main.scss [hunk 4]
```diff
@@ -1,3 +1,67 @@
...
+    border: 1px solid black;
+    background-color: #eee;
+    box-sizing: border-box;
+    cursor: pointer;
+    transition: 0.5s;
+
+    @for $dayNumber from 1 through 31 {
+      &:nth-child(#{$dayNumber}):before {
+        content: "#{$dayNumber}";
+        font-size: 30px;
```
Comment (etojeDenys) [inline src/styles/main.scss]: try to re-deploy, 'cause cannot fund this style in deployed version

**Review by etojeDenys — CHANGES_REQUESTED** [review]

**Pull request closed without merging**